.github/workflows/release-changelog.yml

```diff
@@ -1,6 +1,22 @@
 name: release-changelog
 on: workflow_dispatch
 jobs:
+  get-release-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      id: ${{ steps.id-generator.outputs.id }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: id-generator
+        run: echo "::set-output name=id::$(./bin/create-next-release-tag.sh)"
+  tag-release:
+    runs-on: ubuntu-latest
+    needs: get-release-tag
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./bin/create-release.sh ${{needs.get-release-tag.outputs.id}}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Just in case: there is an action to create a release: https://github.com/softprops/action-gh-release

Maybe it makes sense to use it instead of our scripts.

Smth like:

```
      - name: Release
        uses: softprops/action-gh-release@v0.1.14
        with:
          generate_release_notes: true
          draft: true
```

Reply:
Oh I just tested this in UI. This ROCKS! Will redo it to use that instead.